tests/framework/clients/corral/config.go

```diff
@@ -13,11 +13,14 @@ const (
 // CorralConfigs is a struct that for necessary corral config environment variables to build a corral
 type CorralConfigs struct {
 	CorralConfigVars map[string]string `json:"corralConfigVars" yaml:"corralConfigVars"`
+	CorralConfigUser string            `json:"corralConfigUser" yaml:"corralConfigUser" default:"jenkauto"`
```

Comment:
What's the purpose for CorralConfigUser, is this a new corral feature.

Reply:
> IMO, the most important reason I added these is that the corralConfigUser: <string, default="jenkauto"> user is what is used to name resources created in the backend, which is important for cleanup and correct naming practices we should be updating this to our own shortname (i.e. ctw for me).